src/Sarif.Driver/Sdk/AnalyzeCommandBase.cs

```diff
@@ -30,7 +30,6 @@ public abstract class AnalyzeCommandBase<TContext, TOptions> : PluginDriverComma
         private Run _run;
         private Tool _tool;
         internal TContext _rootContext;
-        private CacheByFileHashLogger _cacheByFileHashLogger;
```

Comment:
>_cacheByFileHashLogger [](http://example.com/codeflow?start=38&length=22)

I've clobbered caching by file hash in general with this change. It will not come back for the single-threaded analysis, as this engine is getting dropped entirely.